netuitive/conf/netuitive-agent.conf

```diff
@@ -139,10 +140,10 @@ include_cpu_pct = True
 enabled = True
 # exclude everything that begins /boot or /mnt
 exclude_filters = ^/boot, ^/mnt
-simple = True
```

Comment:
If a user had this overridden to `False` will the collector flag still overwrite the now global flag?